Rb.Forms.Barcode.Droid/BarcodeScannerRenderer.cs

```diff
@@ -24,7 +25,11 @@ public class BarcodeScannerRenderer : ViewRenderer<BarcodeScanner, SurfaceView>,
         private CameraService CameraService {
             get {
                 if (null == cameraService) {
-                    var configurator = new CameraConfigurator(config, Context);
+                    var configurator = new CameraConfigurator();
+
+                    configurator.SetContext(Context)
+                        .SetConfiguration(config);
+                    
                     var factory = new CameraServiceFactory(configurator);
                     cameraService = factory.Create(Context, Element);                    
```

Comment:
i would like to prevents static as it could lead to unexpected behaviour in the camera scenario like the CameraService is set and the camera open even if this is a new instance of the class.